ethstorage/downloader/blob_cache_test.go

```diff
@@ -7,39 +7,97 @@ import (
 	"bytes"
 	"fmt"
 	"math/big"
+	"math/rand"
 	"os"
+	"path/filepath"
 	"testing"
+	"time"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto/kzg4844"
 	"github.com/ethstorage/go-ethstorage/ethstorage"
+	"github.com/ethstorage/go-ethstorage/ethstorage/log"
 	"github.com/protolambda/go-kzg/eth"
 )
 
 var (
-	bc        BlobCache
+	cache     BlobCache
```

Comment:
This is for the unit test. I think we may need to set up a real test environment to see the results:

1.	Deploy a test storage contract with a shard size of 512 GB.
2.	Upload 3 to 6 blobs every 2 seconds.

Then, we should collect the results, including:

	1.	Ensuring that the memory usage of es-node remains stable.
	2.	The disk-backed cache should be around 16 GB (131072 * 3 * 3600 * 12) just before finalization.
	3.	Monitor how long it takes to delete all the cache items afterward.
	4.	Record the time required to write the 16 GB of data to the disk after finalization, noting any exceptions or issues during this process.
	5.	Monitor the time it takes to close the cache when shutting down es-node.

Reply:
will be done on L2 testnet